### ISSUE TITLE
glm-1.0.1-light identifies itself as version 1.0.0

### ISSUE THREAD

**Issue by ryandesign** [g-truc/glm#1302]
glm-1.0.0-light.7z contains the same files as the glm directory of the 1.0.0 tag of the repository, except for the addition of the copying.txt file.

glm-1.0.1-light.7z contains the same files as the glm directory of the 1.0.1 tag of the repository, except for the addition of the copying.txt file and the fact that `GLM_VERSION_PATCH` is defined to `0` instead of `1` and `GLM_VERSION_MESSAGE` is defined to `GLM: version 1.0.0` instead of `GLM: version 1.0.1`. This discrepancy seems like an error.

**Comment by Guest-Liang** [comment]
Agree, you can see that the upload time for zip and 7z files are Feb 27, but source code upload time is Feb28.
That troubles me for a while.
But basic function still works well, I hope next release can fix this.
